core/templates/dev/head/css/oppia.css

```diff
@@ -2513,7 +2513,7 @@ div#ng-curtain {
   margin-top: 4px;
   padding: 4px;
   resize: none;
-  width: 245px;
+  width: 199px;
```

Comment:
@souravbadami I would advice in the against using `199px` since we would not be in the favor of using magic numbers in our css. IMO a change of 199 to 200 wouldn't be fairly visible, so you might prefer the one which suits better in terms of the code perspective!